src/components/Common/Modal/Modal.tsx

```diff
@@ -30,60 +30,48 @@ Modal.defaultProps = {
 
 export default function Modal(props: ModalProps): JSX.Element {
 	const { t } = useTranslation();
-
-	return props.visible ? (
+	if (!props.visible) return null;
+	
+	return (
```

Comment:
Should be changed to return props.visible && ( Component )

Reply:
See 7adf6b3 in #297 #312